### PR TITLE
Add unit support points for FE_TraceQ

### PR DESCRIPTION
**Pull request by kronbichler**

### CODE REVIEW

File: source/fe/fe_trace.cc [hunk 1]
```diff
@@ -49,6 +49,13 @@ FE_TraceQ<dim,spacedim>::FE_TraceQ (const unsigned int degree)
 
   // Initialize face support points
   this->unit_face_support_points = fe_q.get_unit_face_support_points();
+
+  // initialize unit support points (this makes it possible to assign initial
+  // values to FE_TraceQ)
+  this->unit_support_points.resize(this->dofs_per_cell);
+  for (unsigned int i=0; i<this->dofs_per_cell; ++i)
+    this->unit_support_points[i] = fe_q.get_unit_support_points()[i];
```
Comment (masterleinad) [inline source/fe/fe_trace.cc]: This means that FE_TraceQ has the same unit_support_points as FE_Q, doesn't it?
Is this really what we want? Should it not just be the support_points on the lower-dimensional objects?

Comment (kronbichler) [inline source/fe/fe_trace.cc]: Well, it does leave out the support points associated with the element-local degrees of freedom. For example, for Q3 elements in 2D `this->dofs_per_cell` is 12 and not 16 as in FE_Q, so the last 4 are left out. This is a subtle difference and probably worth a comment.

Comment (masterleinad) [inline source/fe/fe_trace.cc]: Thanks.

**Comment by masterleinad** [comment]
Looks good to me.

**Pull request merged**